### ISSUE TITLE
Multilanguage Login form should switch language too

### ISSUE THREAD

**Issue by lenamtl** [projectsend/projectsend#216]
Multilanguage Login form should switch language too

If the default language is french, then if I select english this should change the form to english 
now it stay in french

**Comment by ignacionelson** [comment]
If you want to change a name, includes/language-locales-names.php has the array.
I really don't know where to begin looking for the real, well translated names of the languages.
Any suggestions?

**Comment by ignacionelson** [comment]
Oh, I got your point. maybe you got the "use brower's language" option enabled, and that's why the form is appearing in french.

**Comment by lenamtl** [comment]
No I have not enable the option to use browser language.
If I select english the form stay in french.
I have tested it under Chrome and IE nothing happens on the form.
Where is the language trigger when switching language on login?

When selecting another language + login, the language is ok on the backend.
the problem occurred only on login form itself

About the language name I have fixed as you suggest 
Just editing the language list seems enough it's working ok for now .
'fr' => "Français",

To reproduce set the default site language other than english let say french and have at least 2 languages in the selector french (default) and english
